config/config.default.js

```diff
@@ -190,7 +190,9 @@ var path = require('path'),
         visualization: {
             decoratorPaths: [path.join(__dirname, '../src/client/decorators')],
             visualizerDescriptors: [path.join(__dirname, '../src/client/js/Visualizers.json')]
-        }
+        },
+
+        layout: 'js/Layouts/DefaultLayout/DefaultLayout'
```

Comment:
Does this mean we need to add custom requirejs paths for layouts? It's probably better do handle them the same way as Plugins and AddOns using basePaths.